integration/tools/validation/src/test/java/alluxio/cli/ValidationTestUtils.java

```diff
@@ -95,8 +94,8 @@ private static Element createElement(Document doc, String k, String v) {
   public static File prepareConfDir() throws IOException {
     // The dir path will contain randomness so will be different every time
     File testConfDir = createTemporaryDirectory();
-    InfoCollectorTestUtils.createFileInDir(testConfDir, "core-site.xml");
-    InfoCollectorTestUtils.createFileInDir(testConfDir, "hdfs-site.xml");
```

Comment:
These two lines were actually a mistake that should be changed to not a util method for other tests. This overlook wasn't identified until I moved this class out of that module.